botorch/utils/transforms.py

```diff
@@ -124,19 +124,52 @@ def normalize_indices(indices: Optional[List[int]], d: int) -> Optional[List[int
     return normalized_indices
 
 
+def _verify_output_shape(cls: Any, X: Tensor, output: Tensor) -> bool:
```

Comment:
If the first argument is an acquisition function object, let's maybe use `acqf: AcquisitionFunction` for this?

Reply:
This actually leads to a circular import